tests/refinement_composition_gradient/screen-output

```diff
@@ -8,7 +8,7 @@ Number of degrees of freedom: 5,734 (2,178+289+1,089+1,089+1,089)
    Solving Bob system ... 0 iterations.
 
    Postprocessing:
-     Writing graphical output:  output-refinement_composition_gradient/solution/solution-00000
+     Writing graphical output:  output/solution/solution-00000
```

Comment:
I think this change is not correct, and expect that the tester is going to fail. Please take the diffs from the tester to update the test output again.